homeassistant/components/husqvarna_automower/entity.py

```diff
@@ -0,0 +1,72 @@
+"""Platform for Husqvarna Automower basic entity."""
+
+import datetime
+import logging
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+from homeassistant.util import dt as dt_util
+
+from . import AutomowerDataUpdateCoordinator
+from .const import DOMAIN, HUSQVARNA_URL
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class AutomowerEntity(CoordinatorEntity[AutomowerDataUpdateCoordinator]):
+    """Defining the Automower Basic Entity."""
+
+    _attr_has_entity_name = True
+    _attr_should_poll = False
+
+    def __init__(self, coordinator, idx) -> None:
```

Comment:
Why do we actually use a number to differentiate on the mowers? Could we maybe use some mower id instead?

Reply:
I thought it's the easiest way, becuase if we poll the API we get a list of mowers. So first mower is 0, second mower is 1 and so on.
Otherwise we would have to search in the dictionary of the mowers for the mower id

Reply:
Can't we map it with mowerid: mower?

Reply:
As it is a dataclass now, it's easy to iterate through it. I think, it's okay now